### PR TITLE
Small fixes, Dutch translation + parameters remaining devices

### PR DESCRIPTION
**Pull request by caseda**
Code made more readable/consistant
Some small fixes/changes.
Dutch translation to all settings that are present now.

### CODE REVIEW

**Comment by caseda** [comment]
I did add the id to association group 4, didn't just bluntly copied it over :).
thanks for the extra heads-up though!

File: drivers/FGD-212/driver.js [hunk 1]
```diff
@@ -69,13 +73,38 @@ module.exports = new ZwaveDriver( path.basename(__dirname), {
 			"index": 20,
 			"size": 1
 		},
+		"double_click_option": {
+			"index": 23,
+			"size": 1,
+			"parser": function( value ){
+				return new Buffer([ ( value === true ) ? 0 : 1 ]);
```
Comment (robertraaijmakers) [inline drivers/FGD-212/driver.js]: Is it me, or should 1 be returned when the value in this parser is true? (same for the other boolean parsers)

Default = true (enabled) and enabled should be value 1

Available settings:
0 - double click disabled
1 - double click enabled
Default setting: 1 Parameter size: 1 [byte]

**Comment by caseda** [comment]
Oh man, that is something that slipped through,
my mistake for blindingly copying @WeeJeWel 's parser code
(does explain why it didn't work when 1.0.6 was introduced)

Will change all checkbox parser :)
Thank you :+1: @robertraaijmakers

File: drivers/FGD-212/driver.js (hunk 1 above)
Comment (caseda) [inline drivers/FGD-212/driver.js]: Oh man, that is something that slipped through,
my mistake for blindingly copying @WeeJeWel 's parser code
(does explain why it didn't work when 1.0.6 was introduced)

Will change all checkbox parser :)
Thank you :+1: @robertraaijmakers

**Comment by caseda** [comment]
Need to change the int parsers to Number parsers, int parses doesn't leave with a proper number value. 
Can't fix it atm (will do so this evening) 

So if something seems of/not working when you change something with numbers, this is the cause.

**Comment by caseda** [comment]
Because some errors that gotten in and its easier to just start over with the (correct) changes, so closing this one

**Pull request closed without merging**